datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupActions.java

```diff
@@ -365,23 +322,23 @@ protected void doExecute(SdxContext context, DatalakeBackupFailedEvent payload,
             protected Object getFailurePayload(DatalakeBackupFailedEvent payload, Optional<SdxContext> flowContext, Exception ex) {
                 return DatalakeDatabaseBackupFailedEvent.from(payload, ex);
             }
-        };
-    }
 
-    private String getFailureReason(Map<Object, Object> variables, Exception exception) {
-        StringBuilder reason = new StringBuilder();
-        if (variables.containsKey(REASON) && variables.get(REASON).equals(DatalakeBackupFailureReason.BACKUP_ON_UPGRADE.name())) {
-            reason.append("Upgrade not started, datalake backup failed.");
-        } else {
-            if (exception instanceof PollerStoppedException) {
-                reason.append("Backup timed out, see the backup status using cdp-cli for more information.");
-            } else {
-                reason.append("Backup failed, returning datalake to running state.");
+            private String getFailureReason(Map<Object, Object> variables, Exception exception) {
+                StringBuilder reason = new StringBuilder();
+                if (variables.containsKey(REASON) && variables.get(REASON).equals(DatalakeBackupFailureReason.BACKUP_ON_UPGRADE.name())) {
+                    reason.append("Upgrade not started, datalake backup failed.");
+                } else {
+                    if (exception instanceof PollerStoppedException) {
+                        reason.append("Backup timed out, see the backup status using cdp-cli for more information.");
+                    } else {
+                        reason.append("Backup failed, returning datalake to running state.");
+                    }
+                }
+                if (exception != null && StringUtils.isNotEmpty(exception.getMessage())) {
+                    reason.append(" Failure message: ").append(exception.getMessage());
+                }
+                return reason.toString();
             }
-        }
-        if (exception != null && StringUtils.isNotEmpty(exception.getMessage())) {
-            reason.append(" Failure message: ").append(exception.getMessage());
-        }
-        return reason.toString();
+        };
     }
 }
```

Comment:
Looks like indentation needs to be corrected.

Reply:
Indentation is fine, I am making this method a subclass of the abstract class for the action since it is not used anywhere else. This isn't needed, just a slight change to clear up one of the syntax warnings thrown by IntelliJ.